hub/test/src/testDrivers/integrationTestDrivers.js

```diff
@@ -0,0 +1,131 @@
+/* @flow */
+
+import fs from 'fs'
+import os from 'os'
+import path from 'path'
+import http from 'http'
+import express from 'express'
+
+import { DriverModel } from '../../../src/server/driverModel'
+import AzDriver from '../../../src/server/drivers/AzDriver'
+import S3Driver from '../../../src/server/drivers/S3Driver'
+import GcDriver from '../../../src/server/drivers/GcDriver'
+import DiskDriver from '../../../src/server/drivers/diskDriver'
+import { InMemoryDriver } from './InMemoryDriver'
+
+/**
+ * Either a:
+ *  - file path to a json file (must end in `.json`)
+ *  - json string
+ *  - base64 encoded json string
+ */
+const driverConfigTestData = process.env.DRIVER_CONFIG_TEST_DATA
+
+const envConfigPaths = { 
+  az: process.env.AZ_CONFIG_PATH, 
+  aws: process.env.AWS_CONFIG_PATH, 
+  gc: process.env.GC_CONFIG_PATH, 
+  disk: process.env.DISK_CONFIG_PATH 
+};
+
+export const driverConfigs = {
+  az: undefined,
+  aws: undefined,
+  gc: undefined,
+  disk: undefined
+};
+
+if (driverConfigTestData) {
```

Comment:
Should we just use the env vars defined by the drivers/config.json itself to pass in the config data?

Reply:
I mostly set it up this way to avoid having to setup lots of env vars locally or in a given CI system in order to perform all integration tests

Reply:
Makes sense